packages/bundle-source/src/index.js

```diff
@@ -55,11 +57,46 @@ export default async function bundleSource(
     if (isEntry) {
       entrypoint = fileName;
     }
+    let unmappedCode = code;
+    if (
+      moduleFormat === 'nestedEvaluate' &&
+      !fileName.startsWith('_virtual/')
+    ) {
+      unmappedCode = '';
+      // eslint-disable-next-line no-await-in-loop
+      const consumer = await new SourceMapConsumer(chunk.map);
```

Comment:
`new SourceMapConsumer` returns a promise? That’s surprising.

Reply:
Yeah, isn't it!

Reply:
(I think it's really just a thenable.)